handlers/github_api_v1/readme.md

```diff
@@ -0,0 +1,33 @@
+# Github API V1
+Github REST API V1 Client
+
+## Parameters
+[Error Handling]
+  Select between returning an error message, or raising an exception.
+[Method]
+  HTTP Method to use for the Kinetic Core API call being made.
+  Options are:
+    - GET
+    - POST
+    - PUT
+    - PATCH
+    - DELETE
+[Path]
+  The relative API path (to the `api_location` info value) that will be called.
+  This value should begin with a forward slash `/`.
+[Body]
+  The body content (JSON) that will be sent for POST, PUT, and PATCH requests.
+
+## Results
+[Response Body]
+  The returned value from the Rest Call (JSON format)
+
```

Comment:
I missed this in my original review - but you are missing the "Response Code" and "Handler Error Message" results.